packages/TelegramClientTests-UI.package/TCTUChatMessageListTests.class/instance/fillWithMessages.st

```diff
@@ -0,0 +1,6 @@
+testing
+fillWithMessages
+
+	self subject addBottom: (TCTMMocks mockMessage message: 'First message').
+	self subject addBottom: (TCTMMocks mockMessage message: 'Second message').
+	self subject addBottom: (TCTMMocks mockMessage message: 'Third message').
```

Comment:
it would be better to use `mockMessageWith: self core` to ensure deletion of the core. Also, why is this not called in `setUp` ?

Reply:
it is not called in setUp because these messages are specific to the test. If we added them in the setup it would impact the other test we have written.